src/helpers/platforms.tsx

```diff
@@ -0,0 +1,12 @@
+import { GoogleMeet } from '@/components/Icons/GoogleMeet';
+
+export const platforms = {
+  'google-meet': {
+    displayName: 'Google Meet',
+    icon: <GoogleMeet />,
+  },
+  teams: {
+    displayName: 'Teams',
+    icon: <GoogleMeet />,
```

Comment:
need other supported platforms icons as svg format

Reply:
Want to double check if this is solved?

Reply:
Yeah, we can add the rest on the next PRs, need the svg files from figma possibly.